cdap-ui/app/cdap/components/FastAction/ExploreAction/ExploreModal.js

```diff
@@ -29,7 +29,7 @@ export default class ExploreModal extends Component {
     super(props);
     this.type = this.props.entity.type === 'datasetinstance' ? 'dataset' : this.props.entity.type;
     this.state = {
-      queryString: 'SELECT * FROM ' + this.type + '_' + this.props.entity.id + ' LIMIT 500',
+      queryString: `SELECT * FROM ${this.props.entity.databaseName}.${this.props.entity.tableName} LIMIT 500`,
```

Comment:
It will still be in the table name. We are now defaulting it to be always `databaseName.tableName`